### PR TITLE
AB#23532 Contract section in Project Info

### PR DESCRIPTION
**Pull request by aurelio-aot**

### CODE REVIEW

File: applications/Unity.GrantManager/src/Unity.GrantManager.Web/Views/Shared/Components/ProjectInfo/Default.cshtml [hunk 1]
```diff
@@ -141,15 +141,31 @@
...
+            <abp-column size="_12" class="d-inline-flex justify-content-between pb-3 pt-2 project-location">
+                <div id="ProjectInfoTitleRegion" class="d-flex align-items-stretch justify-content-start">
+                    <h6 class="ps-1 fw-bold">@L["ProjectInfoView:ProjectInfoContract"].Value</h6>
+                </div>
+            </abp-column>
+        </abp-row>
+        <abp-row class="m-0 project-info-location">
+            <abp-row class="m-0 p-0">
+                <abp-column size="_4" class="px-1">
+                    <abp-input asp-for="@Model.ProjectInfo.ContractNumber" onchange="enableProjectInfoSaveBtn(this)" />
```
Comment (AndreGAot) [inline applications/Unity.GrantManager/src/Unity.GrantManager.Web/Views/Shared/Components/ProjectInfo/Default.cshtml:155]: should we not make this type="number"

Comment (aurelio-aot) [reply applications/Unity.GrantManager/src/Unity.GrantManager.Web/Views/Shared/Components/ProjectInfo/Default.cshtml]: > should we not make this type="number"

hi @AndreGAot -- i have now made the necessary changes.

**Comment by aurelio-aot** [comment]
hi @AndreGAot -- as discussed, Contract # and Execution Date is already set here in the Application entity. thanks.

**Review by AndreGAot — APPROVED** [review]
Ready for Dev

**Pull request merged**